e2e/__tests__/workerForceExit.test.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {tmpdir} from 'os';
+import {resolve} from 'path';
+import findProcess from 'find-process';
+
+import {
+  cleanup,
+  generateTestFilesToForceUsingWorkers,
+  writeFiles,
+} from '../Utils';
+import runJest from '../runJest';
+
+const DIR = resolve(tmpdir(), 'worker-force-exit');
+
+beforeEach(() => cleanup(DIR));
+const testFiles = {
```

Comment:
can you not have this between setup and teardown?

Reply:
I actually thought it made sense because the files are created between setup and teardown 😅 but okay I can change it ^^